cursive/Cargo.toml

```diff
@@ -48,11 +48,33 @@ version = "2"
 optional = true
 version = "0.26"
 
+[dependencies.wasm-bindgen]
+optional = true
+version = "0.2.63"
+
+[dependencies.web-sys]
+optional = true
+version = "0.3.64"
+features = [
+    "HtmlCanvasElement",
+    "CanvasRenderingContext2d",
+    "KeyboardEvent",
+    "TextMetrics",
+    "Window",
+    "Document",
+    "console",
+]
+
+[dependencies.getrandom]
+optional = true
+version = "0.2"
+features = ["js"] 
+
 [features]
 doc-cfg = ["cursive_core/doc-cfg"]  # Enable doc_cfg, a nightly-only doc feature.
 builder = ["cursive_core/builder"]
 blt-backend = ["bear-lib-terminal"]
-default = ["ncurses-backend"]
+default = []
```

Comment:
Any reason to remove the default backend? That just break things for people who don't select one explicitly? (And e.g. running examples from the repository)